Fix: Importing FastStore components's style (#1173)
vtexdocs/dev-portal-content: docs/faststore/components/getting-started.mdx

> ## Adding styles

Reviewer: ```suggestion
## Applying Styles to FastStore UI components
```

> There are two ways to style FastStore UI components:
> 
> - [Overriding a component](https://developers.vtex.com/docs/guides/faststore/overrides-native-component): Customize a native component.
> - [Creating a new section](https://developers.vtex.com/docs/guides/faststore/building-sections-creating-a-new-section): Build a new section for the Headless CMS.
> 
> In both scenarios, you'll create a stylesheet for your component and import the necessary styles. For more details on creating a stylesheet and importing styles, refer to the [Importing FastStore UI component styles](https://developers.vtex.com/docs/guides/faststore/using-themes-importing-ui-components-styles).

Reviewer: ```suggestion
There are two main scenarios where you may want to apply styles to FastStore UI components:

- [Overriding a component](https://developers.vtex.com/docs/guides/faststore/overrides-native-component): Customize the appearance of an existing component.
- [Creating a new section](https://developers.vtex.com/docs/guides/faststore/building-sections-creating-a-new-section): Style a new UI section.

In both scenarios, you'll create a stylesheet for your component and import the necessary styles. For more details on creating a stylesheet and importing styles, refer to the [Importing FastStore UI component styles](https://developers.vtex.com/docs/guides/faststore/using-themes-importing-ui-components-styles).
```

```sh
@import "@faststore/ui/src/styles/base.scss";
```

> In this case, you should also import the desired components' styles individually by checking the component references to import their styles.

Reviewer: ```suggestion
In this case, you should also import the styles for each component individually by referring to the component documentation.
```
